src/com/koxudaxi/ruff/Ruff.kt

```
@@ -173,12 +174,16 @@ val intellijLspClientSupported: Boolean
         }
         return try {
             @Suppress("UnstableApiUsage")
-            LspServerSupportProvider
-            intellijLspClientSupportedValue = true
-            true
+            val supported = LspServerSupportProvider.Companion.EP_NAME
+                .findFirstAssignableExtension(RuffLspServerSupportProvider::class.java) != null
+            intellijLspClientSupportedValue = supported
+            supported
         } catch (e: NoClassDefFoundError) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)